allennlp/tests/fixtures/crf_tagger/experiment.json

```diff
@@ -51,9 +51,11 @@
       "dropout": 0.0,
       "bidirectional": true
     },
-    "regularizer": [
-      ["transitions$", {"type": "l2", "alpha": 0.01}]
-    ]
+    "regularizer": {
+      "regexes": [
```

Comment:
We need to start keeping track of breaking config changes if we aren't going to update them in the models as we go, because otherwise doing them all at once is going to be a nightmare. Can you add this to the release notes also?

Reply:
Yeah, thanks for the reminder, I added it.  I am volunteering to make all of the changes to our saved model archives.  You're right, it will be a pain, but I have the most context on what all of the changes are and how to fix them.